Modules/_cursesmodule.c

```diff
@@ -1217,8 +1221,9 @@ PyCursesWindow_GetStr(PyCursesWindowObject *self, PyObject *args)
         Py_END_ALLOW_THREADS
         break;
     case 1:
-        if (!PyArg_ParseTuple(args,"i;n", &n))
+        if (!PyArg_ParseTuple(args, "i:getstr", &n)) {
```

Comment:
This change looks unrelated to this issue. It will be outdated after converting the module to Argument Clinic.